src/components/RegestrationForm.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+	<v-dialog v-model="isDialog" persistent max-width="600px" min-width="360px">
+		<div>
+			<v-tabs
+				v-model="tab"
+				show-arrows
+				background-color="deep-purple accent-4"
+				icons-and-text
+				dark
+				grow
+			>
+				<v-tabs-slider color="purple darken-4" />
+				<v-tab v-for="i in tabs" :key="i.name">
```

Comment:
what if name will be the same?

Reply:
`tabs: [
			{ name: "Login", icon: "mdi-account" },
			{ name: "Register", icon: "mdi-account-outline" },
		],`
Should I add the ids for tabs?

Reply:
yep

Reply:
you can use **v4** library or create a helper function in utils folder for generation uniq id

Reply:
Okey, but should I use it in simple cases like this one? Or id like "0" or "1" would be enough?